.eslintrc

```diff
@@ -2,6 +2,10 @@
   "extends": "airbnb-base",
   "rules": {
     "linebreak-style": 0,
-    "arrow-body-style":"off"
+    "arrow-body-style":"off",
+    "import/no-unresolved": [0]
+  },
+  "env":{
+    "mocha": true 
   } 
```

Comment:
add this to the `.eslintrc`

`  "settings": {
    "import/extensions": [".js", ".jsx"]
  },`